test/staking/TokenStaking.test.js

```diff
@@ -4542,9 +4337,6 @@ describe("TokenStaking", () => {
             authorizer.address
           )
 
-        await tToken.connect(staker).approve(tokenStaking.address, 1)
-        await tokenStaking.connect(staker).topUp(stakingProvider.address, 1)
-
```

Comment:
What do you think about add something like that:
```         
await tokenStaking.connect(deployer).setMinimumStakeAmount(1)
await tToken.connect(staker).approve(tokenStaking.address, 1)
await tokenStaking.connect(staker).topUp(stakingProvider.address, 1)
```
To show T stake won't help to unstake 
Not necessary of course

Reply:
Makes sense. Addressed in https://github.com/threshold-network/solidity-contracts/pull/74.